js-miniapp-sdk/scripts/docs.sh

```diff
@@ -14,6 +16,12 @@ version: "$PACKAGE_VERSION"
 ---"
 cat <<< "$contents" > "$FILE_VERSION_MD"
 
-npx typedoc --includeVersion --out $DIR_DOCS/api src --plugin typedoc-plugin-markdown --stripInternal --readme none
+# Generate docs
+npx typedoc --includeVersion --out $DIR_DOCS/api src --plugin typedoc-plugin-markdown --stripInternal --readme none --mode file
```

Comment:
I'm not sure but by changing `--readme none` into `--readme index.md`, could it help you to define the parent as `index.html`?

Reply:
I tried something like this, but it didn't seem to work. It still renames the file to `README.md` for some reason.